src/Compilers/Core/Portable/Emit/NoPia/CommonEmbeddedField.cs

```diff
@@ -201,6 +201,10 @@ Cci.ITypeReference Cci.IFieldReference.GetType(EmitContext context)
                 return UnderlyingField.GetType(context);
             }
 
+            ImmutableArray<Cci.ICustomModifier> Cci.IFieldReference.RefCustomModifiers => ImmutableArray<Cci.ICustomModifier>.Empty;
+
+            bool Cci.IFieldReference.IsByReference => false;
+
```

Comment:
I think we shouldn't hardcode the result, but rather follow the pattern that we use for parameters:
```
            bool Cci.IParameterTypeInformation.IsByReference
            {
                get
                {
                    return UnderlyingParameterTypeInformation.IsByReference;
                }
            }

            ImmutableArray<Cci.ICustomModifier> Cci.IParameterTypeInformation.RefCustomModifiers
            {
                get
                {
                    return UnderlyingParameterTypeInformation.RefCustomModifiers;
                }
            }
```